packages/slice-machine/lib/builders/common/Zone/Card/components/Hints/Renderers/vue.js

```diff
@@ -15,6 +13,7 @@ const createDefaultField = (tag = 'span') => (fieldText) => `<${tag}>{{ ${fieldT
 
 const codeByWidgetType = (Widgets) => ({
   [Widgets.ContentRelationship?.CUSTOM_NAME]: createCodeFromTag('todo-hint'),
```

Comment:
It seems like the Hint of the content relationship has not been made everywhere.